reports/pharmacy_first_report.Rmd

```diff
@@ -142,68 +91,103 @@ Links to the codelist for each analysis can be found beneath the relevant sectio
 
 ## Clinical Services
 
-```{r, message=FALSE, warning=FALSE}
-plot_measures(df_measures,
-  title = "Number of consultations for each clinical service per month",
-  measure_names = names(pf_clinical_service_dict),
-  custom_labels = pf_clinical_service_dict,
-  y_label = "Number of codes for consultations",
+### Total population
+
+```{r, message=FALSE, warning=FALSE, fig.height=4, fig.width=4}
+# Select measures and breakdown
+df_measures_selected <- df_measures %>%
+  filter(measure_desc == "clinical_service") %>%
+  filter(is.na(group_by))
+
+# Create visualisation
+plot_measures(
+  df_measures_selected,
+  select_value = denominator,
+  select_interval_date = interval_end,
+  colour_var = NULL,
+  guide_nrow = 1,
+  facet_wrap = FALSE,
+  facet_var = NULL,
 )
 ```
 
-### Clinical Services by Age
-
-```{r, message=FALSE, warning=FALSE}
-
-plot_measures(df_measures,
-  title = "Number of consultations for each clinical service by age per month",
-  measure_names = names(pf_clinical_service_dict_by_age),
-  custom_labels = pf_clinical_service_dict_by_age,
-  y_label = "Number of codes for consultations",
-  facet_var = "age_band",
-  rotate_x_labels = TRUE
+### Breakdown by age
+
+```{r, message=FALSE, warning=FALSE, fig.height=8, fig.width=8}
+# Select measures and breakdown
+df_measures_selected <- df_measures %>%
+  filter(measure_desc == "clinical_service") %>%
+  filter(group_by == "Age")
+
+# Create visualisation
+plot_measures(
+  df_measures_selected,
+  select_value = denominator,
```

Comment:
This should be numerator instead, right? If so, same applies for all the others.

Reply:
Ohhh yes, you're right! Good spot

Reply:
all changed to numerator in 7b19a7c5659e54cb9bff33ea051a6d716c789258